scalafix-tests/input/src/main/scala/test/escapeHatch/AnchorOverlapping.scala

```diff
@@ -0,0 +1,185 @@
+/*
+rules = [
+  "class:scalafix.test.NoDummy"
+  "class:scalafix.test.NoNull"
+]
+*/
+package test.escapeHatch
+
+object AnchorOverlapping {
```

Comment:
Can we add a test suite named `Issue694` to reproduce the bug from https://github.com/scalacenter/scalafix/issues/694 ?

```scala
  val map: mutable.Map /* scalafix:ok */ [String, Data] = mutable.Map.empty /* scalafix:ok */
```

That would be a good sanity check to make sure the bug is fixed.

Reply:
done